### ISSUE TITLE
adapt-contrib-spoor: Drop _requireCourseCompleted and _requireAssessmentPassed properties

### ISSUE THREAD

**Issue by brian-learningpool** [adaptlearning/adapt_framework#2001]
For version 3.0 these properties have moved to config.json:

The defaults are shown below:
```
_completionCriteria: {
  _requireContentCompleted: true,
  _requireAssessmentPassed: false
}
```

**Comment by moloko** [comment]
as discussed in the gitter chat all the logic is now in core so it should only be necessary for spoor to listen to the `'tracking:complete'` event now

**Comment by brian-learningpool** [comment]
The defaults should actually be:

```
_completionCriteria: {
  _requireContentCompleted: true,
  _requireAssessmentCompleted: false
}
```

This is corrected in 35a8f2d.
